apps/trustlab/src/payload/collections/Partners.js

```diff
@@ -0,0 +1,65 @@
+import {
+  image,
+  richText,
+  socialLinks,
+  nestCollectionUnderPage,
+  slug,
+} from "@commons-ui/payload";
+
+const Partners = {
+  slug: "partners",
+  labels: {
+    singular: {
+      en: "Partner",
+    },
+    plural: {
+      en: "Partners",
+    },
+  },
+  admin: {
+    defaultColumns: ["name", "updatedAt"],
+    enableRichTextLink: false,
+    group: "Organisation",
```

Comment:
```suggestion
    group: "Consortium",
```

or Project since this site does not belong to a single organisation